lib/std/json/write_stream.zig

```diff
@@ -148,6 +148,7 @@ pub fn WriteStream(comptime OutStream: type, comptime max_depth: usize) type {
             self.popState();
         }
 
+        // TODO better handling of ComptimeInt and ComptimeFloat
```

Comment:
`comptime_int` is quite easy to handle, just cast it to the type returned by `math.IntFittingRange(value, value)` and you can treat it as an int type.